src/components/templates/Audience/Audience.tsx

```diff
@@ -410,31 +413,35 @@ export const Audience: React.FunctionComponent = () => {
           style={{ backgroundImage: `url(${venue.mapBackgroundImageUrl})` }}
         >
           <div className="audience">
-            <div
-              ref={focusElementOnLoad}
-              className="video-container"
-              style={videoContainerStyles}
-            >
-              <div className="video">
-                <iframe
-                  className={videoFrameClasses}
-                  src={iframeUrl}
-                  title="Video"
-                  frameBorder="0"
-                  allow={IFRAME_ALLOW}
-                  allowFullScreen
-                />
-              </div>
-
-              {venue.showReactions && (
-                <div
-                  className={`reaction-container ${userSeated ? "seated" : ""}`}
-                >
-                  {userSeated
-                    ? renderReactionsContainer()
-                    : renderInstructions()}
+            <div className="video-container-wrapper" style={videoWrapperStyles}>
+              <div
+                ref={focusElementOnLoad}
+                className="video-container"
+                style={videoContainerStyles}
+              >
+                <div className="video">
+                  <iframe
+                    className={videoFrameClasses}
+                    src={iframeUrl}
+                    title="Video"
+                    frameBorder="0"
+                    allow={IFRAME_ALLOW}
+                    allowFullScreen
+                  />
                 </div>
-              )}
+
+                {venue.showReactions && (
+                  <div
+                    className={`reaction-container ${
+                      userSeated ? "seated" : ""
+                    }`}
```

Comment:
Would be better to use the `classNames` helper here.

Reply:
[OOS] but addded in https://github.com/sparkletown/sparkle/pull/1237/commits/6ff3b5876858d4e306936f59a383e5f0e9c734aa